src/pages/[platform]/build-a-backend/data/custom-business-logic/search-and-aggregate-queries/index.mdx

```diff
@@ -148,13 +148,24 @@ const openSearchDomain = new opensearch.Domain(
   {
     version: opensearch.EngineVersion.OPENSEARCH_2_11,
     nodeToNodeEncryption: true,
+    // set removalPolicy to DESTROY to make sure the open search domain is deleted on stack deletion.
+    removalPolicy: RemovalPolicy.DESTROY,
     encryptionAtRest: {
       enabled: true
     }
   }
 );
 // highlight-end
 ```
+<Callout>
+
+Important considerations:
+
+We recommend enabling the `removalPolicy` for sandbox environments.
+By default, OpenSearch instances are not deleted when you run `npx ampx delete`, as the default removal policy is set to retain the resource.
+
+</Callout>
```

Comment:
```suggestion
<Callout warning>

**Important considerations:**

We recommend configuring the `removalPolicy` to destroy resources for sandbox environments. By default, OpenSearch instances are not deleted when you run `npx ampx sandbox delete`, as the default removal policy for stateful resources is set to retain the resource.

</Callout>
```

let's pivot to a warning to make the callout a bit more prominent since this has pricing implications